src/errors/test-run/templates.js

```diff
@@ -84,8 +93,11 @@ export default {
     `),
 
     [TYPE.uncaughtErrorOnPage]: err => markup(err, `
-        Error on page <a href="${err.pageDestUrl}">${err.pageDestUrl}</a>:
+        A JavaScript error occurred on ${formatUrl(err.pageDestUrl)}.
+        If this error occurs when TestCafe is not used, you can turn the --skip-js-errors option on.
```

Comment:
I prefer the more detailed message with step-by-step instructions.
We usually provide the detailed instructions in issues.

```js
A JavaScript error occurred on ${formatUrl(err.pageDestUrl)}.
At this moment, TestCafe tracks uncaught JavaScript errors on the page.
Try to manually perform the test scenario. If this error still occurrs then it means you site has uncaught JavaScript errors. To disable JavaScript error tracking you can turn the --skip-js-errors option on.
If the error occurs only with TestCafe then it`s a bug. Write a new issue about it at: ...
```

Reply:
I agree, your version is good

Reply:
I checked your version with Paul